google/cloud/pubsub/samples/samples.cc

```diff
@@ -29,26 +29,12 @@
 #include <utility>
 
 namespace {
-std::string RandomTopicId(google::cloud::internal::DefaultPRNG& generator) {
-  return google::cloud::pubsub_testing::RandomTopicId(generator,
-                                                      "cloud-cpp-samples");
-}
-
-std::string RandomSubscriptionId(
-    google::cloud::internal::DefaultPRNG& generator) {
-  return google::cloud::pubsub_testing::RandomSubscriptionId(
-      generator, "cloud-cpp-samples");
-}
 
-std::string RandomSnapshotId(google::cloud::internal::DefaultPRNG& generator) {
-  return google::cloud::pubsub_testing::RandomSnapshotId(generator,
-                                                         "cloud-cpp-samples");
-}
-
-std::string RandomSchemaId(google::cloud::internal::DefaultPRNG& generator) {
-  return google::cloud::pubsub_testing::RandomSchemaId(generator,
-                                                       "cloud-cpp-samples");
-}
+using ::google::cloud::pubsub::examples::RandomSchemaId;
+using ::google::cloud::pubsub::examples::RandomSnapshotId;
+using ::google::cloud::pubsub::examples::RandomSubscriptionId;
+using ::google::cloud::pubsub::examples::RandomTopicId;
+using ::google::cloud::pubsub::examples::UsingEmulator;
```

Comment:
I assume your plan is to simplify these:
https://github.com/googleapis/google-cloud-cpp/blob/fe127b8ae0f2b92635fa414a6f548efaa5cf0770/google/cloud/pubsub/samples/samples.cc#L1883
https://github.com/googleapis/google-cloud-cpp/blob/fe127b8ae0f2b92635fa414a6f548efaa5cf0770/google/cloud/pubsub/samples/samples.cc#L1931